src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -19,6 +19,7 @@
 using Uno.Extensions.Specialized;
 using System.Collections;
 using System.Linq;
+using System.Reflection;
```

Comment:
not needed ?

Reply:
No longer (split in a dedicated file), will remove